src/scss/packages/usa-image-example/src/_styles.scss

```diff
@@ -0,0 +1,41 @@
+@use 'uswds-core' as *;
+
+@mixin usa-image-example($color) {
+  img {
+    box-shadow: 0 units(1) 0 0 color($color);
+  }
+  .usa-icon-list__icon {
+    color: color($color);
+  }
+  .usa-icon-list__content {
+    .image-example__do-dont {
+      color: color($color);
+      font-weight: bold;
+    }
+  }
+}
+
+.usa-image-example__figure {
+  margin: 0;
+  display: table;
+}
+
+.usa-image-example__figcaption {
+  margin-top: units(1.5);
+  display: table-caption;
+  caption-side: bottom;
+}
+
+.usa-image-example__img--bordered {
```

Comment:
When an image has a lighter background we want to lightly border the top, left, and right sides. We don't want this border when the image has a darker background.

For naming this, I tried to follow: https://en.bem.info/methodology/naming-convention/
Specifically, I looked at this example: `menu__item_visible` which in "two dashes style" ([link](https://en.bem.info/methodology/naming-convention/#two-dashes-style)) would end up being: `menu__item--visible` 

So I think I ended up with:
- block-name: `usa-image-example`
- elem-name: `image` (currently `img` but fixing now)
- mod-name: `bordered`

Reply:
Screenshots in this [comment](https://github.com/18F/identity-design-system/pull/369#discussion_r1303095251) (on this PR).